core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2516,4 +2516,14 @@ package object config {
       .version("3.5.0")
       .booleanConf
       .createWithDefault(false)
+
+  private[spark] val DECOMMISSION_MAX_RATIO =
+    ConfigBuilder("spark.decommission.maxRatio")
+      .doc("Max ratio of decommissioning executor of running executors. Decommission too many " +
+        s"executors at the same time with shuffle or rdd migration enabled " +
```

Comment:
If this is the condition of perf regression, we should not use `maxRatio` when data migration (shuffle and RDD) is not enabled.

Reply:
Updated to only use this when data migration (shuffle and RDD) is not enabled.